content/en/tracing/trace_collection/custom_instrumentation/php/otel.md

```diff
@@ -17,31 +17,99 @@ further_reading:
     text: 'Interoperability of OpenTelemetry API and Datadog instrumented traces'
 ---
 
-{{% otel-custom-instrumentation %}}
+{{% otel-custom-instrumentation-lang %}}
 
-## Requirements and limitations
+## Setup
 
-- Datadog PHP tracing library `dd-trace-php` version 0.94.0 or greater.
+To configure OpenTelemetry to use the Datadog trace provider:
 
-The following OpenTelemetry features implemented in the Datadog library as noted:
-
-| Feature                                | Support notes                                           |
-|----------------------------------------|---------------------------------------------------------|
-| [OpenTelemetry Context propagation][1] | [W3C Trace Context and Datadog header formats][9] are enabled by default. |
-| [Span limits][2]                       | Unsupported                                             |
-| [Metrics API][7]                       | Unsupported                                             |
-| Trace/span [ID generators][3]          | ID generation is performed by the tracing library, with support for [128-bit trace IDs][12].    |
-
-## Configuring OpenTelemetry to use the Datadog tracing library
-
-1. Add your desired manual OpenTelemetry instrumentation to your PHP code following the [OpenTelemetry PHP Manual Instrumentation documentation][5]. **Important!** Where those instructions indicate that your code should call the OpenTelemetry SDK, call the Datadog tracing library instead.
+1. Install [OpenTelemetry API packages][13].
+  ```php
+  composer require open-telemetry/sdk
+  ```
+1. Add your desired manual OpenTelemetry instrumentation to your PHP code following the [OpenTelemetry PHP Manual Instrumentation documentation][5]. 
```

Comment:
I think the numbering should continue here, so this line would be 2. and the remaining steps here should be incremented by one